app/javascript/src/components/Dashboard/Notes/NoteList.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "@bigbinary/neeto-icons";
```

Comment:
Please go through the Slack thread very carefully. I had already added a review comment for the same.